### PR TITLE
Mitigate mutes that don't expire when they should

### PR DESCRIPTION
**Pull request by panpawn**

### CODE REVIEW

File: rooms.js [hunk 1]
```diff
@@ -127,7 +127,12 @@ class Room {
 				if (user.userid === entry.userid ||
 					user.guestNum === entry.guestNum ||
 					(user.autoconfirmed && user.autoconfirmed === entry.autoconfirmed)) {
-					return entry.userid;
+					if (entry.time < 0) {
```
Comment (Zarel) [inline rooms.js]: Why would this ever be less than 0?

Comment (panpawn) [reply rooms.js]: Right

Comment (panpawn) [reply rooms.js]: Updated

**Comment by Zarel** [comment]
This is a mitigation, not a fix.

**Comment by Zarel** [comment]
Which is fine but the commit message should be adjusted accordingly.

**Comment by panpawn** [comment]
Adjusted the commit message

**Comment by Zarel** [comment]
I think PS uses "Workaround" but that's also subtly different.

**Pull request merged**